compiler-plugin/build.gradle

```diff
@@ -3,13 +3,13 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'com.github.johnrengelman.shadow'
 
 dependencies {
-    compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
May I ask, why we had to change it to `comple`?

Reply:
Of course @i-walker , questions are welcome!! Because I changed it to `compileOnly` without looking at `shadowJar` configuration where `compile` dependencies are included:

```
configurations = [project.configurations.compile]
```

Reply:
Meaning the compiler-jar now includes the std lib?